slo-monitor/go.mod

```diff
@@ -1,15 +1,9 @@
 module k8s.io/perf-tests/slo-monitor
 
 require (
-	bitbucket.org/ww/goautoneg v0.0.0-20120707110453-75cd24fc2f2c // indirect
-	github.com/beorn7/perks v0.0.0-20160229213445-3ac7bf7a47d1 // indirect
 	github.com/golang/glog v0.0.0-20141105023935-44145f04b68c
 	github.com/googleapis/gnostic v0.2.0 // indirect
-	github.com/matttproud/golang_protobuf_extensions v0.0.0-20150406173934-fc2b8d3a73c4 // indirect
-	github.com/prometheus/client_golang v0.0.0-20151008143247-e51041b3fa41
-	github.com/prometheus/client_model v0.0.0-20150212101744-fa8ad6fec335 // indirect
-	github.com/prometheus/common v0.0.0-20151014055050-ffe929a3f4c4 // indirect
-	github.com/prometheus/procfs v0.0.0-20150928173926-454a56f35412 // indirect
```

Comment:
Shall we remove those guys from the `vendor` subdirectory on the occasion?

Reply:
Yes - you need to regenerate vendor/ (or get rid of it completely).

Reply:
Yes, I think you need to run go mod tidy and go mod vendor here.

We should remove vendor as well, but maybe in a separate PR?

Reply:
I'll get rid of it in a follow-up PR.

Reply:
Sure - it shouldn't be the same PR. It should be either before this PR or after.

Reply:
Done.